packages/engine/src/scene/systems/SceneObjectSystem.ts

```diff
@@ -50,6 +50,11 @@ export default async function SceneObjectSystem(world: World): Promise<System> {
 
       ;(object3DComponent.value as any).entity = entity
 
+      // Enable scene layer
+      object3DComponent.value.traverse((obj: Object3D) => {
```

Comment:
setting scene layer by default will cause all objects to be overridden - this has been observed in the golf game with a lot of weirdness due to custom avatars. rather than setting it here, we should set it whereever these objects are created.